assignments/assifnment7/faulty-oops.md

```diff
@@ -0,0 +1,156 @@
+workingset: timestamp_bits=46 max_order=15 bucket_order=0
+fuse: init (API version 7.37)
+Block layer SCSI generic (bsg) driver version 0.4 loaded (major 249)
+io scheduler mq-deadline registered
+io scheduler kyber registered
+pci-host-generic 4010000000.pcie: host bridge /pcie@10000000 ranges:
+pci-host-generic 4010000000.pcie:       IO 0x003eff0000..0x003effffff -> 0x0000000000
+pci-host-generic 4010000000.pcie:      MEM 0x0010000000..0x003efeffff -> 0x0010000000
+pci-host-generic 4010000000.pcie:      MEM 0x8000000000..0xffffffffff -> 0x8000000000
+pci-host-generic 4010000000.pcie: Memory resource size exceeds max for 32 bits
+pci-host-generic 4010000000.pcie: ECAM at [mem 0x4010000000-0x401fffffff] for [bus 00-ff]
+pci-host-generic 4010000000.pcie: PCI host bridge to bus 0000:00
+pci_bus 0000:00: root bus resource [bus 00-ff]
+pci_bus 0000:00: root bus resource [io  0x0000-0xffff]
+pci_bus 0000:00: root bus resource [mem 0x10000000-0x3efeffff]
+pci_bus 0000:00: root bus resource [mem 0x8000000000-0xffffffffff]
+pci 0000:00:00.0: [1b36:0008] type 00 class 0x060000
+pci 0000:00:01.0: [1af4:1005] type 00 class 0x00ff00
+pci 0000:00:01.0: reg 0x10: [io  0x0000-0x001f]
+pci 0000:00:01.0: reg 0x14: [mem 0x00000000-0x00000fff]
+pci 0000:00:01.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
+pci 0000:00:01.0: BAR 4: assigned [mem 0x8000000000-0x8000003fff 64bit pref]
+pci 0000:00:01.0: BAR 1: assigned [mem 0x10000000-0x10000fff]
+pci 0000:00:01.0: BAR 0: assigned [io  0x1000-0x101f]
+virtio-pci 0000:00:01.0: enabling device (0000 -> 0003)
+cacheinfo: Unable to detect cache hierarchy for CPU 0
+virtio_blk virtio0: 1/0/0 default/read/poll queues
+virtio_blk virtio0: [vda] 122880 512-byte logical blocks (62.9 MB/60.0 MiB)
+rtc-pl031 9010000.pl031: registered as rtc0
+rtc-pl031 9010000.pl031: setting system clock to 2025-10-12T20:18:54 UTC (1760300334)
+NET: Registered PF_INET6 protocol family
+Segment Routing with IPv6
+In-situ OAM (IOAM) with IPv6
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+NET: Registered PF_PACKET protocol family
+NET: Registered PF_KEY protocol family
+NET: Registered PF_VSOCK protocol family
+registered taskstats version 1
+EXT4-fs (vda): INFO: recovery required on readonly filesystem
+EXT4-fs (vda): write access will be enabled during recovery
+EXT4-fs (vda): orphan cleanup on readonly fs
+EXT4-fs (vda): recovery complete
+EXT4-fs (vda): mounted filesystem with ordered data mode. Quota mode: disabled.
+VFS: Mounted root (ext4 filesystem) readonly on device 254:0.
+devtmpfs: mounted
+Freeing unused kernel memory: 1280K
+Run /sbin/init as init process
+  with arguments:
+    /sbin/init
+  with environment:
+    HOME=/
+    TERM=linux
+EXT4-fs (vda): re-mounted. Quota mode: disabled.
+scull: loading out-of-tree module taints kernel.
+scullsingle registered at f800008
+sculluid registered at f800009
+scullwuid registered at f80000a
+scullpriv registered at f80000b
+Hello, world from VrushabhGadaCU!
```

Comment:
I don't see an invocation of the faulty module, or any analysis of the resulting oops here.